tests/test_scxml.py

```diff
@@ -0,0 +1,33 @@
+import xml.etree.ElementTree as ET
+from typing import Optional, Dict, List
+import json
+import pytest
+from xstate.scxml import scxml_to_machine
+
+test_dir = "node_modules/@scion-scxml/test-framework/test"
+
+test_groups: Dict[str, List[str]] = {"actionSend": ["send1"]}
+
+test_files = [
+    (f"{test_dir}/{k}/{vv}.scxml", f"{test_dir}/{k}/{vv}.json")
+    for k, v in test_groups.items()
+    for vv in v
+]
+
+
+@pytest.mark.parametrize("scxml_source,scxml_test_source", test_files)
```

Comment:
This lets us dynamically execute a ton of tests, because we're going to add more. I had to look this up, basically a copy-paste.